src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -5489,26 +5489,26 @@ public void StartEvent(string eventName, EventAttribute eventAttribute)
             byteArrArgIndices = null;
 
             events.Append("  <event").
-                 Append(" value=\"").Append(eventAttribute.EventId).Append("\"").
-                 Append(" version=\"").Append(eventAttribute.Version).Append("\"").
-                 Append(" level=\"").Append(GetLevelName(eventAttribute.Level)).Append("\"").
-                 Append(" symbol=\"").Append(eventName).Append("\"");
+                 Append(" value=\"").Append(eventAttribute.EventId).Append('"').
```

Comment:
this isn't caught by https://github.com/dotnet/roslyn-analyzers/pull/3610 you just noticed it?

Reply:
Oh, I see your comment.